### PR TITLE
Add description for compiler packages that are pre-release

### PR DESCRIPTION
**Pull request by agocke**

### CODE REVIEW

File: src/NuGet/BuildNuGets.csx [hunk 1]
```diff
@@ -143,16 +143,23 @@ string[] TestPackageNames = {
 // or they will not be published anywhere at all
 var PreReleaseOnlyPackages = new HashSet<string>
 {
+    // Contains the compiler build task, but Microsoft.NET.Compilers
+    // should be preferred. Used to construct a standalone compiler for
+    // dotnet CLI.
     "Microsoft.CodeAnalysis.Build.Tasks",
```
Comment (jasonmalinowski) [inline src/NuGet/BuildNuGets.csx]: Add blank lines so it's clear which lines your comments apply to?

File: src/NuGet/BuildNuGets.csx [hunk 2]
```diff
@@ -143,16 +143,23 @@ string[] TestPackageNames = {
 // or they will not be published anywhere at all
 var PreReleaseOnlyPackages = new HashSet<string>
 {
+    // Contains the compiler build task, but Microsoft.NET.Compilers
```
Comment (jcouv) [inline src/NuGet/BuildNuGets.csx]: Should such comments go into the package descriptions themselves, so they are visible on nuget?
Also consider updating our documentation page for packages: https://github.com/dotnet/roslyn/wiki/NuGet-packages

Comment (agocke) [reply src/NuGet/BuildNuGets.csx]: These are comments why these packages *aren't* being uploaded to NuGet.org, so I don't think the package description is really the right place to put them. There's some disagreement about people where these comments to go, so I'm going to put them here for now and anyone who wants to copy the info to another place can do so. 😄

Comment (jcouv) [reply src/NuGet/BuildNuGets.csx]: Ok, makes sense.

**Comment by agocke** [comment]
@jasonmalinowski I believe I've addressed your comments. Anything else?

**Pull request merged**